styles/trusted-customer.css

```diff
@@ -0,0 +1,60 @@
+.trusted-customer__slider {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 60px;
+}
+.trusted-customer__item {
+  border: 2px solid #dddddd;
+  border-radius: 10px;
+  padding: 30px;
+  max-width: 400px;
+  width: 100%;
+  position: relative;
+}
```

Comment:
It would be a nice thing to add style for the user image class `trusted-customer__riezh-robert` with a border-radius: 50% — in future (potential future of this website) no one will crop people photos so that they are circles by default. It will be square (best option) pictures, or rectangle, which must look good in this UI. So thinking ahead is always a good idea.

Reply:
++